agenta-web/src/components/Evaluations/Evaluations.tsx

```diff
@@ -501,9 +507,12 @@ export default function Evaluations() {
             </div>
             <EvaluationErrorModal
                 isModalOpen={!!evaluationError}
-                onClose={() => setEvaluationError("")}
-                handleNavigate={() => router.push(`/apps/${appName}/testsets`)}
+                closeEvalutionError={setEvaluationError}
```

Comment:
Instead of passing state setters as props, reset them in onClose:

`onClose={()=> setError({ message: "", btnText: "", endpoint: "" })}`